[Refactor] Move tutorials from Blog to Tutorial section

rajarshimaitra opened the pull request:
This is a move only PR and moves all the tutorial materials from blog posts to tutorial segment.

I believe we have enough tutorials right now to make sense to keep them separate from regular blog post sections..

I made the move of the files but the webpage is not showing the articles in the tutorial segment..

rajarshimaitra committed:
Move tutorials from Blog to Tutorial section

rajarshimaitra commented:
@ConorOkus @notmandatory Does this require some more changes to correctly build??

ConorOkus commented:
Hmm try running `npm install` locally and committing again.

ConorOkus commented:
Remember to update the links as well - https://github.com/bitcoindevkit/bitcoindevkit.org/blob/master/docs/.vuepress/config.js#L44

rajarshimaitra commented:
> Remember to update the links as well

Yup.. That's what I missed.. Fixed..

rajarshimaitra commented:
- Removed the old hello world blog.
- Moved the Descriptor based paper wallet creation demo into tutorial from blog..

rajarshimaitra committed:
Update Tutorial names

After #103 the tutorial title comes in the side bar in the website,
its better to have shorter names.

ConorOkus commented on docs/.vuepress/config.js:
```diff
@@ -47,6 +47,12 @@ const tutorialSidebar = [
       '/tutorials/hello-world',
+      '/tutorials/Bitcoin_Core_RPC_Demo',
+      '/tutorials/compact_filters_demo',
+      '/tutorials/descriptors_in_the_wild',
```
This page needs a title.

ConorOkus replied:
Last 4 articles need titles actually.

ConorOkus replied:
@rajarshimaitra I mean page heading, these start with a paragraphs

rajarshimaitra committed:
Remove permalinks

It seems they aren't used anymore and pointing to previous locations.

rajarshimaitra commented:
Added title to the Hello World piece.. 

Also removed the `permalink` field. Seemed they were pointing to previous locations and not used anymore?? Let me know if thats not ok..

rajarshimaitra committed:
add article headings and make heading consistent

This adds `## Introduction` heading to all tutorial articles and
makes heading sizes consistent across all tutorials..

rajarshimaitra commented:
> @rajarshimaitra I mean page heading, these start with a paragraphs

Ahh I missed those.. Added in the last push.. Also made the heading sizes consistent across tutorials..

rajarshimaitra commented:
Added changes of #105 into this PR.

rajarshimaitra commented:
Rebased on master..
